include/GafferImage/ImagePlug.h

```diff
@@ -135,6 +136,39 @@ class ImagePlug : public Gaffer::ValuePlug
 		IECore::MurmurHash imageHash() const;
 		//@}
 
+		/// @name Other convenience accessors
+		/// For things that should never depend on channelName and tileOrigin
```

Comment:
We should document the reason for these accessors, providing the guideline that they should be used when accessing from a per-tile compute/hash, but not from a compute/hash that is already in a global context.